rasa/cli/data.py

```diff
@@ -229,6 +259,64 @@ def split_nlu_data(args: argparse.Namespace) -> None:
     telemetry.track_data_split(args.training_fraction, "nlu")
 
 
+def augment_nlu_data(args: argparse.Namespace) -> None:
+    """Performs data augmentation for NLU.
+
+    Data augmentation works by determining intents with low performance or little
+    data from the existing training data and the generated paraphrases are used as
+    additional training data for these intents.
+
+    Subsequently, two models with augmented data are trained and then evaluated on
+    the supplied test data.
+
+    The module outputs a report highlighting how data augmentation has changed
+    classification performance.
+
+    Args:
+        args: Command-line arguments.
+    """
+    rasa.shared.utils.common.mark_as_experimental_feature(
+        "Data augmentation by paraphrasing for NLU"
+    )
+
+    # Collect inputs
+    nlu_training_data = rasa.shared.nlu.training_data.loading.load_data(
+        args.nlu_training_data
+    )
+    nlu_evaluation_data = rasa.shared.nlu.training_data.loading.load_data(
+        args.nlu_evaluation_data
+    )
+    paraphrases = rasa.shared.nlu.training_data.loading.load_data(args.paraphrases)
+
+    default_report_file_path = os.path.join(
+        DEFAULT_RESULTS_PATH, INTENT_REPORT_FILE_NAME
+    )
+    report_file = rasa.cli.utils.get_validated_path(
+        args.nlu_classification_report, "", default_report_file_path
+    )
+    classification_report = rasa.shared.utils.io.read_json_file(report_file)
+
+    if args.augmentation_factor <= 0.0:
+        raise ValueError("The argument 'augmentation_factor' must be greater than 0!")
```

Comment:
should in my opinion be some subclass of `RasaException` so that exceptions of this type don't get forwarded to Sentry

Reply:
changed to `InvalidParameterException` which seems appropriate for this case. lmk if this is unsuitable for cmdline args.